[2022-08-20 07:39:41,848] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data scheduled__2022-08-19T18:00:00+00:00 [queued]>
[2022-08-20 07:39:42,899] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: yandex-rates-test.insert_data scheduled__2022-08-19T18:00:00+00:00 [queued]>
[2022-08-20 07:39:42,942] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-20 07:39:42,964] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-20 07:39:42,986] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-20 07:39:44,520] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): insert_data> on 2022-08-19 18:00:00+00:00
[2022-08-20 07:39:44,863] {standard_task_runner.py:52} INFO - Started process 4619 to run task
[2022-08-20 07:39:45,104] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'yandex-rates-test', 'insert_data', 'scheduled__2022-08-19T18:00:00+00:00', '--job-id', '35', '--raw', '--subdir', 'DAGS_FOLDER/yandex_dag.py', '--cfg-path', '/tmp/tmpejlnsb1b', '--error-file', '/tmp/tmp55bwm9a1']
[2022-08-20 07:39:45,122] {standard_task_runner.py:80} INFO - Job 35: Subtask insert_data
[2022-08-20 07:39:49,269] {task_command.py:371} INFO - Running <TaskInstance: yandex-rates-test.insert_data scheduled__2022-08-19T18:00:00+00:00 [running]> on host 00f41d9230c6
[2022-08-20 07:39:52,732] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=yandex-rates-test
AIRFLOW_CTX_TASK_ID=insert_data
AIRFLOW_CTX_EXECUTION_DATE=2022-08-19T18:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-19T18:00:00+00:00
[2022-08-20 07:39:53,269] {logging_mixin.py:115} INFO - rate_date:  2022-08-20
[2022-08-20 07:39:53,311] {logging_mixin.py:115} INFO - value_:  20922.3125
[2022-08-20 07:39:53,523] {base.py:68} INFO - Using connection ID 'postgre_test_db' for task execution.
[2022-08-20 07:39:53,808] {python.py:173} INFO - Done. Returned value was: None
[2022-08-20 07:40:01,962] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=yandex-rates-test, task_id=insert_data, execution_date=20220819T180000, start_date=20220820T073941, end_date=20220820T074001
[2022-08-20 07:40:08,999] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-20 10:21:40,211] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
